tests/jupyter/test_pythreejs.py

```diff
@@ -221,9 +232,8 @@ def test_labels():
     pl = pyvista.Plotter()
     pl.add_point_labels(poly, "My Labels", point_size=20, font_size=36)
 
-    # ensure that we ignore labels
-    with pytest.warns(UserWarning):
-        pv_pythreejs.convert_plotter(pl)
+    # ensure point labels at least don't raise a warning
+    pv_pythreejs.convert_plotter(pl)
```

Comment:
Is this change related to the PR? And do warnings get converted to errors in CI?

Reply:
This is related to this PR and warnings do not get converted into errors. Fixing that...